docs/source/concepts/pipeline_schema/index.rst

```diff
@@ -484,4 +484,256 @@ Data dependencies *between* these steps are removed, and then the step nodes are
+
+Entity resolution pipeline schema
+---------------------------------
+
+.. image:: images/entity_resolution_pipeline_schema.drawio.png
+
+Input datasets
+^^^^^^^^^^^^^^
+
+**Interpretation:**
+A set of named datasets.
+Each dataset contains observations recorded about (some) entities in the population of interest for analysis.
+
+**Specification:**
+A directory of files, where each file is in a tabular format.
+Each file's name identifies the name of that input dataset.
+Each file may have any number of columns,
+but one of them must be called “Record ID” and it must have unique values.
+
+**Example:**
+
+.. list-table:: 
+   :header-rows: 1
+
+   * - Record ID
+     - First
+     - Last
+     - Address
+   * - 1
+     - Vicki
+     - Simmons
+     - 123 Main St. Apt C, Anytown WA 99999
+   * - 2
+     - Gerald
+     - Allen
+     - 456 Other Drive, Anytown WA, 99999
+
+Clusters
+^^^^^^^^
+
+**Interpretation:**
+A (partial) clustering of the input records,
+which indicates that records assigned the same cluster ID are observations of the same entity
+and records with different cluster IDs are observations of different entities.
+Records without a cluster ID are unresolved
+(they may or may not be part of one of the existing clusters).
+Pre-assigned clusters can be used to indicate any prior knowledge of clustering.
+If there is no prior knowledge, this can be an empty table (which is the default),
+indicating that all records are unresolved.
+
+**Specification:**
+A file in a tabular format with two columns: "Input Record ID" and "Cluster ID".
+"Input Record ID" must have unique values,
+each of which is the combination of a dataset name and a Record ID value found in the corresponding input dataset.
+"Cluster ID" may take any value.
+
+**Example:**
+
+.. list-table:: 
+   :header-rows: 1
+
+   * - Input Record ID
+     - Cluster ID
+   * - input_file_1
+     - 1
+   * - input_file_2
+     - 2
+   * - reference_file_1
+     - 2
+   * - input_file_4
+     - 3
+   * - input_file_5
+     - 3
+   * - reference_file_2
+     - 3
+
+In this example, record ID 1 from the dataset "input_file" has been put in its own cluster,
+meaning that it does not match any of the other records listed.
+input_file_2 has been put in a cluster with reference_file_1,
+indicating that they refer to the same person.
+input_file_3 doesn't appear in the table at all, meaning that its cluster is unknown.
+Lastly, input_file_4 and input_file_5 are considered duplicates
+(records, from the same data source, referring to the same entity)
+and are also a match to reference_file_2.
+
+Clustering pass
+^^^^^^^^^^^^^^^
+
+**Interpretation:**
+A pass at clustering (some) records.
+This may take into account already-found clusters as it sees fit:
+anything from using them as a starting point for optimization to treating those clusters as set-in-stone and unchangeable.
+
+Going from one of these passes to the next is
+one kind of *cascading*, in which a consistent clustering
```

Comment:
Add a link? (even if to an empty doc that will be filled in later)

Reply:
Chose to include more info inline, and changed the TODO to be organizational (possibly splitting it out to a different page)